cmf-cli/Commands/new/IoTCommand.cs

```diff
@@ -253,9 +236,32 @@ public void ExecuteV10AngularPackage(IDirectoryInfo workingDir, string version,
                 ForceColorOutput = false
             }.Exec();
 
-            // Install IoT Yeoman
+            InstallYoeman(iotCustomPackageWorkDir, mesVersion);
+
+            #region Link To HTML Package
+
+            // After building the IoT Package we must ensure to also build the HTML Package
+            iotCustomPackage.RelatedPackages = new()
+            {
+                new RelatedPackage() { Path = fileSystem.Path.GetRelativePath(iotCustomPackageWorkDir.FullName, htmlPackageDir.FullName).Replace("\\", "/"), PostBuild = true, PostPack = true }
+            };
+
+            iotCustomPackage.SaveCmfPackage();
+
+            #endregion Link To HTML Package
+        }
+
+        private static void InstallYoeman(IDirectoryInfo iotCustomPackageWorkDir, Version mesVersion)
+        {
             Log.Debug($"Installing Yeoman");
 
```

Comment:
do we need to do this? in the handler, you are invoking yeoman directly from the CLI installation directory, why do you need it installed here? Library scaffolding?